src/server/common/model/answer/date/date-utils.test.js

```diff
@@ -28,41 +27,52 @@ describe('Date Helpers', () => {
   })
 
   describe('isFutureDate', () => {
+    beforeEach(() => {
+      jest.useFakeTimers()
+      jest.setSystemTime(new TZDate('2025-04-01T00:00:00', 'Europe/London'))
```

Comment:
If we are only using TZDate in tests you might want to move that package to devDependencies

Reply:
We still have one usage in `toBSTDate` in the implementation - great thing to check though :+1: